osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -0,0 +1,256 @@
+import os
+import shutil
+import subprocess
+import tempfile
+import time
+import sys
+import traceback
+
+from celery import shared_task
+
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+from django.core.files import File
+from django.core.files.storage import FileSystemStorage
+from django.contrib import messages
+
+from excerptconverter.baseexcerptconverter import BaseExcerptConverter
+from excerptconverter import ConverterHelper
+
+from osmaxx.excerptexport import models
+
+
+private_storage = FileSystemStorage(location=settings.PRIVATE_MEDIA_ROOT)
+
+
+class GisExcerptConverter(BaseExcerptConverter):
+    @staticmethod
+    def name():
+        return 'GIS'
+
+    @staticmethod
+    def export_formats():
+        return {
+            'spatialite': {
+                'name': 'SpatiaLite (SQLite)',
+                'file_extension': 'sqlite',
+                'mime_type': 'application/x-sqlite3'
+            },
+            'gpkg': {
+                'name': 'Geo package',
+                'file_extension': 'gpkg',
+                'mime_type': 'application/octet-stream'  # Not verified!
+            },
+            'shp': {
+                'name': 'Shape file',
+                'file_extension': 'shp',
+                'mime_type': 'application/octet-stream'  # Not verified!
+            }
+        }
+
+    @staticmethod
+    def export_options():
+        return {
+            # has to be implemented next:
+            # 'coordinate_reference_system': {
+            #     'label': 'Coordinate reference system',
+            #     'type': 'choice',
+            #     'default': 'pseudomerkator',
+            #     'groups': [
+            #         {
+            #             'name': 'Global coordinate reference systems',
+            #             'values': [
+            #                 {'name': 'pseudomerkator', 'label': 'Pseudo merkator'},
+            #                 {'name': 'wgs72', 'label': 'WGS 72'},
+            #                 {'name': 'wgs84', 'label': 'WGS 84'}
+            #             ]
+            #         },
+            #         {
+            #             'name': 'UTM zones for your export',
+            #             'values': [
+            #                 {'name': 'utm32', 'label': 'UTM zone 32'},
+            #                 {'name': 'utm33', 'label': 'UTM zone 33'}
+            #             ]
+            #         }
+            #     ]
+            # },
+            # 'detail_level': {
+            #     'label': 'Detail level',
+            #     'type': 'choice',
+            #     'default': 'verbatim',
+            #     'values': [
+            #         {'name': 'verbatim', 'label': 'Verbatim'},
+            #         {'name': 'simplified', 'label': 'Simplified'}
+            #     ]
+            # }
+        }
```

Comment:
Currently, no options (apart from the export formats from `export_formats()`) are displayed. The commented-out configuration is for future use.